dev/local/install.sh

```diff
@@ -43,3 +46,6 @@ cd ../../
 
 ln -s repos/delphi/delphi-epidata/dev/local/Makefile
 ln -s repos/delphi/delphi-epidata/dev/local/.dockerignore
+cd repos
+ln -s delphi/delphi-epidata/dev/local/pyproject.toml
+ln -s delphi/delphi-epidata/dev/local/setup.cfg
```

Comment:
```suggestion
ln -s delphi/delphi-epidata/dev/local/setup.cfg
cd -
```

Make the install script back out to the original. Should we install the package here too or is that assuming too much for the user?

Reply:
re:installing the package for the user - I'd suggest not going for that, since
- we can't really guess what environment they want the package installed into - `python` vs `python3` vs `/path/to/python3` vs `/path/to/my/venv` etc.
- it requires a fairly recent version of setuptools, so we'd have to `install --upgrade` that for the user too